spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/ImportExportControllerHelperService.java

```diff
@@ -359,15 +359,23 @@ public JSONObject confirmLibrariesPoolsUpload(HttpSession session, JSONObject js
                 try {
                   LibraryQC lqc = new LibraryQCImpl();
                   lqc.setLibrary(library);
-                  lqc.setInsertSize(insertSize);
                   lqc.setResults(Double.valueOf(jsonArrayElement.getString(5)));
                   lqc.setQcCreator(user.getLoginName());
                   lqc.setQcDate(new Date());
                   lqc.setQcType(requestManager.getLibraryQcTypeByName("Qubit"));
 
+                  LibraryQC lis = new LibraryQCImpl();
+                  lis.setLibrary(library);
+                  lis.setResults((double) insertSize);
+                  lis.setQcCreator(user.getLoginName());
+                  lis.setQcDate(new Date());
+                  lis.setQcType(requestManager.getLibraryQcTypeByName("Insert Size"));
```

Comment:
Our lab isn't using this, but this would fail if we tried -- our insert size QCType is called "Tape Station" because that's the name of the instrument, and we haven't got an "Insert Size" QCType in our database. I'm less fond of the option to rename Tape Station to Insert Size because if we get a new instrument to measure insert size, it doesn't capture the difference.